docs/source/install.rst

```diff
@@ -11,30 +11,35 @@ Here are two ways to hack on python.org:
 Easy setup using Vagrant
 ------------------------
 
+First, install install Vagrant_ and Ansible_ on your machine. 
+You should then be able to provision the Vagrant box
+
 ::
 
     $ vagrant up
-    $ vagrant ssh
-    # go to pythondotorg/ directory and activate virtualenv, then run
-    $ ./manage.py runserver 0.0.0.0:8000
-    # on your local shell
-    $ google-chrome http://localhost:8001/
+    
+The box will be provisioned with Python 3.5, a virtualenv with all
+the requirements installed, and a database ready to use.
 
-The box will be provisioned by Ansible_ 2.4.1.0 with Python 3.5, a virtualenv
-set up with requirements installed, and a database ready to use.
-
-The box also creates a superuser with username ``cbiggles`` for you. However, you
-will need to set a password before using it::
+Once this is done it's time to create some data and run the server
 
+::
+    
     $ vagrant ssh
+    # go to the pythondotorg/ directory and activate virtualenv
```

Comment:
While we're here, can we also use the commenting style described at https://nedbatchelder.com/blog/201401/comments_should_be_sentences.html?